### PR TITLE
multiple fixes and better client debugging

### PR DESCRIPTION
**Pull request by fscherf**
This PR fixes page titles on daemonized views and adds a proper debug mode for the client

### CODE REVIEW

**Comment by fscherf** [comment]
@maratori I had problems ``tests/test_static_files_widget.py`` failing again (this is the test we both debugged before because it failed randomly running in the CI). It seems it had something to do with multi-threading. I fixed it by implementing a hook using PEP 487 (https://www.python.org/dev/peps/pep-0487/)

File: lona/client_pre_compiler.py [hunk 1]
```diff
@@ -48,33 +69,65 @@ def get_enums(self) -> Dict[str, Dict[str, Any]]:
...
+                    source_file,
+                )
 
-            template_context = {
-                'protocol': dumps(self.get_enums()),
-                'settings': dumps(self.get_settings()),
-            }
+                with open(path, 'w+') as f:
+                    f.write(file_content)
+                    f.close()
```
Comment (pandrey2003) [inline lona/client_pre_compiler.py:113]: This line of code may be redundant as the file will be closed by the context manager in `__exit__`

Comment (fscherf) [reply lona/client_pre_compiler.py]: Ack! :) fixed!

Comment (pandrey2003) [reply lona/client_pre_compiler.py]: Happy to have helped you :)
Just a fresh look

Comment (fscherf) [reply lona/client_pre_compiler.py]: Review is always welcome!

**Review by maratori — APPROVED** [review]
LGTM

**Comment by fscherf** [comment]
@maratori: Thanks for reviewing :)

**Pull request merged**